test/parallel/test-crypto.js

```diff
@@ -141,3 +141,8 @@ assert.throws(function() {
 
 // Make sure memory isn't released before being returned
 console.log(crypto.randomBytes(16));
+
+assert.throws(function() {
+      common.fail();
```

Comment:
Thank you. (That was actually something that I should have removed before committing.)